recipe/meta.yaml

```diff
@@ -1,26 +1,27 @@
-{% set version = "3.3" %}
+{% set version = "3.4" %}
 
 package:
   name: idna
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/idna/idna-{{ version }}.tar.gz
-  sha256: 9d643ff0a55b762d5cdb124b8eaa99c66322e2157b69160bc32796e824360e6d
+  sha256: 814f528e8dead7d329833b91c5faa87d60bf71824cd12a7530b5526063d02cb4
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vvv
+  skip: True  # [py<36]
 
 requirements:
   host:
+    - flit-core
     - python
     - pip
     - wheel
     - setuptools
   run:
-    - python >=3.5
+    - python
 
 test:
   imports:
```

Comment:
Please remove `python <3.10` from `test/requires `to unblock the tests for **python 3.10**. It was a temporary fix.

Reply:
Removed the python from `test/requires`.

I was curious, but where did you find this?

Reply:
There was an issue with python 3.10 linking by `conda` when **3.10** was interpreted as **3.1**. That's why we skip testing on that moment.